src/tokens/TokenImport.tsx

```diff
@@ -0,0 +1,193 @@
+                    onPress={() => ValoraAnalytics.track(AssetsEvents.tap_import_token_paste)}
+                  />
+                }
+                maxLength={42} // 0x prefix and 20 bytes
+              />
+            </View>
+
+            {/* Token Symbol */}
+            <View style={styles.textInputGroup}>
+              <Text style={styles.label}>{t('tokenImport.tokenSymbol')}</Text>
+              <TextInput
+                onChangeText={setTokenSymbol}
+                value={tokenSymbol}
+                multiline={false}
+                style={styles.messageTextInput}
+                placeholderTextColor={Colors.gray4}
+                underlineColorAndroid="transparent"
+                numberOfLines={1}
+                showClearButton={true}
+                editable={isValidAddress}
+                right={isValidAddress && <GreenLoadingSpinner height={32} />} // TODO RET-892: once loaded, hide the spinner
+              />
+              {errorMessage()}
+            </View>
+
+            {/* Network */}
+            <View style={styles.textInputGroup}>
+              <Text style={styles.label}>{t('tokenImport.network')}</Text>
+              <TextInput
+                onChangeText={setTokenSymbol}
+                value={t('celoGold') ?? undefined}
+                multiline={false}
+                style={styles.messageTextInput}
+                placeholderTextColor={Colors.gray4}
+                underlineColorAndroid="transparent"
+                numberOfLines={1}
+                showClearButton={true}
+                editable={false}
+              />
+            </View>
+          </View>
+        </View>
+      </ScrollView>
+      <Button
+        size={BtnSizes.FULL}
+        text={t('tokenImport.importButton')}
+        showLoading={false}
+        disabled={tokenSymbol.length == 0} // TODO RET-892: enable button if the contract was loaded
+        onPress={navigateBackAndToast}
+        style={styles.buttonContainer}
+      />
+    </SafeAreaView>
+  )
+}
+
+TokenImportScreen.navigationOptions = {
+  ...noHeader,
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+  },
+  customHeader: {
+    paddingHorizontal: variables.contentPadding,
+  },
+  headerTitle: {
+    ...typeScale.bodyMedium,
+  },
+  messageTextInput: {
+    width: '100%',
+    ...typeScale.bodySmall,
+    paddingHorizontal: Spacing.Small12,
+    borderColor: Colors.gray2,
+    borderRadius: Spacing.Tiny4,
+    borderWidth: 1.5,
+    color: Colors.dark,
+  },
+  titleContainer: {
+    alignItems: 'center',
+    marginVertical: Spacing.Smallest8,
+    marginHorizontal: Spacing.Thick24,
+    gap: Spacing.Regular16,
+    ...typeScale.titleLarge,
+  },
+  buttonContainer: {
+    paddingVertical: Spacing.Regular16,
+    paddingHorizontal: Spacing.Thick24,
+    borderTopColor: Colors.gray2,
+  },
+  textInputGroup: {
+    flexDirection: 'column',
+    justifyContent: 'flex-start',
+    alignItems: 'flex-start',
```

Comment:
flex-start is the default value, do you need these?

Reply:
I believe you meant for `justifyContent`. Correct, I'll remove.

`alignItems`'s default is `stretch` ([docs](https://reactnative.dev/docs/flexbox#align-items))